clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt

```diff
@@ -22,12 +22,8 @@ package org.ossreviewtoolkit.clients.fossid.model.result
 import com.fasterxml.jackson.annotation.JsonProperty
 
 enum class MatchType {
```

Comment:
Please explain in the commit message *why* they are unneeded.

Reply:
> They are unneeded because the SNAKE_CASE mapping strategy from Jackson maps them already.

Hmm, is that rationale really correct? Because you sometimes have lowercase, sometimes capitalized property names in the annotations. IIUC the annotations are not unneeded right *now*, but the will be *will* be when also taking the third commit into account. So IMO you should squash first and third commit.

Reply:
Done.
Please review the commit message: I didn't find out how to write it in imperative form.